### DISCUSSION TITLE
Few improvements for notyKT (backend)

### DISCUSSION THREAD

**Discussion by pipeline1987** [PatilShreyas/NotyKT#564] [Discuss noty-api (Backend development)]
Hi, simple questions and/or proposals:

- have you considered storing note `tags` in the Note.kt model as jsonb structure to show how to handle and passing by across the service to know how to work with this kind of structure data, available in databases engines such posgresql.

- have you considered handling migrations models with flyway.

**Comment by PatilShreyas** [comment]
@pipeline1987 

- We don't see any use case for storing "tags" for notes.
- We are using [JetBrains Exposed](https://github.com/JetBrains/Exposed) which automatically migrates if the model is changed. So currently, it's not needed.
